src/main/webapp/availability.html

```diff
@@ -29,11 +30,11 @@
             <a class="nav-item nav-link" href="index.html">Home</a>
             <!--TODO: Put in links to other pages. Put on all pages-->
             <a class="nav-item nav-link" href="profile.html">Profile</a>
-            <a class="nav-item nav-link" href="scheduled-interviews">Events</a>
+            <a class="nav-item nav-link" href="scheduled-interviews.html">Events</a>
             <a class="nav-item nav-link active" href="#">Availability</a>
             <a class="nav-item nav-link" href="#">Scheduling</a>
             <!--TODO: Move this to the right of the navbar and allow user to log in and out using Authentication-->
```

Comment:
Is this TODO now done? If so, please remove it.

Reply:
Done